### ISSUE TITLE
Password authentication rejected in the Mininet Environment

### ISSUE THREAD

**Issue by PigossiJr** [wichert/pyrad#52]
Hello dear friend, how are you? I am trying to understand what's happening with my code, but I didnt understand why this error is occuring..

Anyway, I am running the pyrad code (.py) in a mininet environment. In the first case, I was able to authenticate my host but, in the second, with the same code, there is a difference in the password authentication (something related to encrypt).

**1) First case (ok - pyrad is running ok)**
Host 1 (h1) -> Radius Server

At the same h1, I tried to authenticate with my python code and everything works perfectly
![1 - ok](https://cloud.githubusercontent.com/assets/25137551/24840097/27468254-1d3d-11e7-9491-b49f5569a1eb.png)

**2) Second case (nok)**
Host 1 (h1) -> Radius Server
Host 2 (h2) -> Trying to authenticate in H1

![2](https://cloud.githubusercontent.com/assets/25137551/24840130/e1acfef2-1d3d-11e7-9ba4-d125f9e5a380.png)

Basically the only difference, comparing the test 1 with test 2, was that: in the test 1, both code and radius server instance was running in the same host. On the other hand, in the test 2, host 1 is running my radius server instance and host 2 is trying to authenticate in host 1.

Ps: I am developing a SDN application based on Rytu controller. I can assure that the conectivity between host1 and host 2 is ok.

I am also attaching my pyrad code here -> 
[auth.zip](https://github.com/wichert/pyrad/files/908467/auth.zip)

I am looking forward your answer and let me know if you need something else.

Thank you!

Best regards,

Sincerely,
Antonio
